cli/QuickstepCli.cpp

```diff
@@ -1,6 +1,6 @@
 /**
  *   Copyright 2011-2015 Quickstep Technologies LLC.
- *   Copyright 2015-2016 Pivotal Software, Inc.
+ *   Copyright 2015 Pivotal Software, Inc.
```

Comment:
Why deleted some copyright? Please reset it.

Also, you should add the university one.